src/plugins_types_bits.c

```diff
@@ -40,19 +40,19 @@ ly_type_store_bits(const struct ly_ctx *ctx, const struct lysc_type *type, const
         const struct lysc_node *UNUSED(ctx_node), struct lyd_value *storage, struct lys_glob_unres *UNUSED(unres),
         struct ly_err_item **err)
 {
+
+    /* necessary variables */
     LY_ERR ret = LY_SUCCESS;
-    size_t item_len;
-    const char *item;
-    struct ly_set *items = NULL, *items_ordered = NULL;
-    size_t buf_size = 0;
-    char *buf = NULL;
-    size_t index;
-    LY_ARRAY_COUNT_TYPE u;
     struct lysc_type_bits *type_bits = (struct lysc_type_bits *)type;
-    ly_bool iscanonical = 1;
-    size_t ws_count;
-    size_t lws_count; /* leading whitespace count */
-    const char *can = NULL;
+    unsigned int index;
```

Comment:
Please do not use `unsigned int`, `uint32_t` is the better alternative. Also, if there is no special reason, put declarations of same-type variables together.